python/src/puml/gen_puml.aac

```diff
@@ -0,0 +1,103 @@
+            A path to a YAML file containing an AaC-defined usecase from which
+            to generate a PlantUML sequence diagram.
+        - name: output-directory
+          type: directory
+          description: |
+            The output directory into which the PlantUML (.puml) diagram file
+            will be written.
+      acceptance:
+        name: Sequence Diagram Test
+        scenarios:
+          - name: Output PlantUML sequence diagram for specified input and output path.
+            given:
+              - The {{puml.input.architecture-file}} contains a valid model.
+              - The positional argument {{puml.input.output-directory}} contains a valid file path.
+            when:
+              - The aac app is run with the puml-sequence command.
+            then:
+              - A PlantUML sequence diagram of the use case is written to the output directory.
+    - name: puml-object
+      help_text: Convert an AaC-defined system to PlantUML object diagram.
+      input:
+        - name: architecture-file
+          type: file
+          description: |
+            A path to a YAML file containing an AaC-defined system from which to
+            generate a PlantUML object diagram.
+        - name: output-directory
+          type: directory
+          description: |
+            The output directory into which the PlantUML (.puml) diagram file
+            will be written.
+      acceptance:
+        name: Object Diagram Test
+        scenarios:
+          - name: Output PlantUML object diagram for specified input and output path.
+            given:
+              - The {{puml.input.architecture-file}} contains a valid model.
+              - The positional argument {{puml.input.output-directory}} contains a valid file path.
+            when:
+              - The aac app is run with the puml-object command.
+            then:
+              - A PlantUML object diagram of the model is written to the output directory.
+    - name: puml-requirements
+      help_text: Convert an AaC-defined system to a requirements diagram in PlantUML format.
+      input:
+        - name: architecture-file
+          type: file
+          description: |
+            A path to a YAML file containing an AaC-defined system from which to
+            generate a PlantUML requirements diagram.
+        - name: output-directory
+          type: directory
+          description: |
+            The output directory into which the PlantUML (.puml) diagram file
+            will be written.
+      acceptance:
+        name: Requirement Diagram Test
+        scenarios:
+        - name: Output PlantUML requirements diagram for specified input file and output path.
+          given:
+            - The {{puml.input.architecture-file}} contains a valid model.
+            - The positional argument {{puml.input.output-directory}} contains a valid file path.
+          when:
+            - The aac app is run with the puml-requirement command.
```

Comment:
You marked as resolved and said it was applied but is still not a match for the command name for this section. 

```suggestion
            - The aac app is run with the puml-requirements command.
```